azure/Guidelines.md

```diff
@@ -0,0 +1,259 @@
+# Microsoft Azure REST Design Guidelines
+
+## History
+
+<dl>
+<dd>2020-Mar-31 v3.0</dd>
+<dt>Version 3.0 of the Azure REST API Guidelines was copied from Sharepoint
+into Markdown and uploaded to GitHub as the basis for future improvements.</dt>
+</dl>
+
+## Introduction
+
+The Azure REST API guidelines are an extension of the Microsoft-wide [OneAPI guidelines][1] (which historically drew heavily from an earlier version of the Azure REST API guidelines). Readers of this document are assumed to be also reading the [OneAPI guidelines][1] and be familiar with them.  Azure guidance is a superset of OneAPI guidelines and services should follow them *except* where this document outlines specific differences or exceptions to those guidelines. This document does contain additional Azure-specific guidance and additional details.
+
+The OneAPI guidelines are available on Github at [https://github.com/microsoft/api-guidelines][1].
+
+Highlights of the OneAPI guidelines include:
```

Comment:
This whole section is redundant now they share a repo - suggest completely remove it.

Reply:
Agreed,